python/mooseutils/unit_tests/tests

```diff
@@ -0,0 +1,32 @@
+[Tests]
```

Comment:
In a comment you mention that you had to run a python blah blah command to run the tests, but doesn't this file plug it into the ability to use run_tests?